### DISCUSSION TITLE
Menu

### DISCUSSION THREAD

**Discussion by Tech-404** [Nurutomo/wabot-aq#686] [Q&A]
Bang di menunya tambahin fake reply, biar keren 🗿

**Comment by rizxyu** [comment]
P ngabs

**Reply by unx21** [reply]
> P ngabs

p

**Reply by SanzGantengz** [reply]
> P ngabs

P

**Comment by fariz-ind** [comment]
cara nampilin menu gimana dh

**Reply by unx21** [reply]
> cara nampilin menu gimana dh

Ketik #menu udah tuh

**Comment by Alganeril** [comment]
.help
